Strategies/ITVision.md

```diff
@@ -0,0 +1,168 @@
+
+The following is ESDC's IT Vision.
+It paints a vision of the future of IT within ESDC.
+It is **not** meant to be time bound, but rather as an ideal state to which ESDC IT needs to strive to get to.
+Such state will continuously evolve when new information and knowledge is captured, re-adjusting the vision in the process.
+
+The IT Vision complements the [HDLC](HumanDevelopmentLifeCycle.md) story which is taken from the point of view of the citizen, and the [KPIs](IITB_KPIs.md) which define the performance metrics needed to sustain the HDLC, by providing the point of view of the organization's IT capabilities.
+
+The IT Vision leverages the work done in the [IITB Way Forward](http://dialogue/grp/IITB-DGIIT-Gov-New-Nouveau/Documents/Departmental%20IMIT%20Plan/19-20%20Plans/IITB%20Moving%20Forward%20v2.docx), ensures alignment with [IITB's high level priorities](http://esdc.prv/en/iitb/corporate/Who_We_Are/Plans_and_Priorities/index.shtml), and brings the [GC's Digital Operations Strategic Plan 2018-2022](https://www.canada.ca/en/government/system/digital-government/digital-operations-strategic-plan-2018-2022.html) into ESDC's operational context.
+
+The ESDC IT Vision is explained using the [GC's Digital Operations Strategic Plan 2018-2022](https://www.canada.ca/en/government/system/digital-government/digital-operations-strategic-plan-2018-2022.html) six chapters:
+
+1. [User-centred, service orientation](#user-centred-service-orientation)
+2. [An open, collaborative, and accessible government](#an-open-collaborative-and-accessible-government)
+3. [Digital-first and digitally enabled by design](#digital-first-and-digitally-enabled-by-design)
+4. [Modern tech, technology practices and information stewardship](#modern-tech-technology-practices-and-information-stewardship)
+5. [A digitally enabled public service](#a-digitally-enabled-public-service)
+6. [Good digital governance](#good-digital-governance)
+
+## 1. User-centred, service orientation
+
+Human Interfaces that citizens interact with (e.g. websites, web applications, and mobile applications) are rarely provided by ESDC.
+Instead, **ESDC does not compete with the private sector** and seeks to leverage it to drastically improve citizen's user experience as well as [promoting Canadian innovation and skills](https://www.theglobeandmail.com/business/commentary/article-the-essentials-for-canadas-digital-economy/).
+ESDC makes use of the public and private industries such as job promoting websites, banks, academic institutions, and employers, to offer its services via APIs (see [An open, collaborative, and accessible government](#an-open-collaborative-and-accessible-government)).
+ESDC establishes terms of service and conditions with private industries to assure its services are always equitable, accessible, and do not have barriers for users with disabilities.
```

Comment:
I recommend to add the privacy consideration. Otherwise, private sector will profit from these services by mining/selling citizen data.